File: Haskell/beginner/1050.hs
-- https://judge.beecrowd.com/pt/problems/view/1050
-- Getting printf to print some awesome things
import Text.Printf

-- Main body of our magic code
main = do
    -- Requesting values and assign into variables
    ddd <- readLn :: IO Int

    -- Testing DDD to find our city
    case ddd of
        61 -> printf "Brasilia\n"
        71 -> printf "Salvador\n"
        11 -> printf "Sao Paulo\n"
        21 -> printf "Rio de Janeiro\n"
        32 -> printf "Juiz de Fora\n"
        19 -> printf "Campinas\n"
        27 -> printf "Vitoria\n"
        31 -> printf "Belo Horizonte\n"
        _  -> printf "DDD nao cadastrado\n"

-- By Fernando Serra
-- https://github.com/fernandosserra

File: C/beginner/1050.c
// https://judge.beecrowd.com/pt/problems/view/1050
// DDD

// Libraries
#include <stdio.h>

// Main Function
int main(){
    // Variables
    int number;

    // Requesting DDD to user
    scanf("%d", &number);

    // Testing conditions and showing results
    switch (number)
    {
    case 61:
        printf("Brasilia\n");
        break;

    case 71:
        printf("Salvador\n");
        break;

    case 11:
        printf("Sao Paulo\n");
        break;

    case 21:
        printf("Rio de Janeiro\n");
        break;

    case 32:
        printf("Juiz de Fora\n");
        break;

    case 19:
        printf("Campinas\n");
        break;

    case 27:
        printf("Vitoria\n");
        break;

    case 31:
        printf("Belo Horizonte\n");
        break;  
    
    // If DDD is not on the list
    default:
        printf("DDD nao cadastrado\n");
        break;
    }

    // Ending code
    return 0;
}

/*
By Fernando Serra
https://github.com/fernandosserra
*/